packages/client/hmi-client/src/components/workflow/ops/calibrate-ensemble-ciemss/calibrate-ensemble-ciemss-operation.ts

```diff
@@ -1,18 +1,47 @@
+import { CiemssMethodOptions } from '@/services/models/simulation-service';
+import { CiemssPresetTypes } from '@/types/common';
 import { Operation, WorkflowOperationTypes, BaseState } from '@/types/workflow';
-import type { EnsembleModelConfigs } from '@/types/Types';
 import calibrateEnsembleCiemss from '@assets/svg/operator-images/calibrate-ensemble-probabilistic.svg';
 
 const DOCUMENTATION_URL = 'https://github.com/ciemss/pyciemss/blob/main/pyciemss/interfaces.py#L156';
 
+export const speedPreset = Object.freeze({
+	numSamples: 1,
+	method: CiemssMethodOptions.euler,
+	numIterations: 10,
+	learningRate: 0.1
+});
+
+export const qualityPreset = Object.freeze({
+	numSamples: 10,
+	method: CiemssMethodOptions.dopri5,
+	numIterations: 100,
+	learningRate: 0.03
+});
 export interface EnsembleCalibrateExtraCiemss {
-	solverMethod: string;
 	numParticles: number; // The number of particles to use for the inference algorithm. https://github.com/ciemss/pyciemss/blob/1fc62b0d4b0870ca992514ad7a9b7a09a175ce44/pyciemss/interfaces.py#L225
+	presetType: CiemssPresetTypes;
+	solverMethod: CiemssMethodOptions;
 	numIterations: number;
+	endTime: number;
+	stepSize: number;
+	learningRate: number;
+}
+
+export interface CalibrateEnsembleMappingRow {
+	name: string;
```

Comment:
If you look at @Tom-Szendrey `SimulateEnsembleMappingRow` he has it as `newName`. See about settling for similar naming between the operators for consistency

Reply:
@Tom-Szendrey I've switch to using newName for consistency sake